Vic2ToHoI4/Source/HOI4World/Decisions/Decision.cpp

```diff
@@ -21,6 +21,10 @@ HoI4::decision::decision(std::string decisionName, std::istream& theStream): nam
 		const commonItems::stringOfItem theAllowed(theStream);
 		allowed = theAllowed.getString();
 	});
+	registerKeyword("highlight_states", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
You shouldn't double the code which you have below.
```registerRegex("highlight_state_targets|highlight_states", ...```
will work well for both cases